### PR TITLE
:sparkles: MostAffectedCaseTable - # mutations col

### PR DESCRIPTION
**Pull request by cy**
~changing this to one query per row~

### CODE REVIEW

**Comment by cy** [comment]
changed to pre-generated fragments, can't even get a simple string interpolation like $ssmFilter${number} to work inside Relay.QL :/ all fragments written out. Looks dumb, but works. Ready for review @NCI-GDC/oicrdevs

File: modules/node_modules/@ncigdc/containers/MostAffectedCasesTable.js [hunk 1]
```diff
@@ -146,6 +167,183 @@ const MostAffectedCasesTableComponent = compose(
...
+  row18: hits(filters: $ssmsFilters38) {
+    total
+  }
+  row19: hits(filters: $ssmsFilters39) {
+    total
+  }
+}
+`;
+
+const SsmVariables = {
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/containers/MostAffectedCasesTable.js]: definitely not a good idea, but you could generate this list from relay
```
   mutationsFragment40.children.reduce((vars, row) => [...vars, ...row.calls.map(call => call.value.callVariableName)], [])
```
would give you an array of the variable names...
but don't do it haha probably pretty fragile.

Comment (alex-wilmer) [inline modules/node_modules/@ncigdc/containers/MostAffectedCasesTable.js]: hmm couldn't this be

```
_.range(0, 40).reduce((p, c) => ({ ...p, [`ssmsFilters${c}`]: null ]), {})
```

**Comment by cy** [comment]
ok, ok `key={caseId}` on the `<MutationsCount>`s made the page change work. Thanks for the help @alex-wilmer. I still don't like it but I'll live.

**Pull request merged**